games/static/games/proto.js

```diff
@@ -263,6 +288,10 @@ class Board {
       .map((square) => square.index + this.direction);
   }
 
+  noAliensLeft() {
```

Comment:
its not recommended to name things negetivly 
its better to name it aliensLeft and just use the ! to negate it:
```js
if (!this.aliensLeft()) { ... }
```